kong/client.go

```diff
@@ -357,6 +357,22 @@ func (c *Client) Status(ctx context.Context) (*Status, error) {
 	return &s, nil
 }
 
+// Config returns the config of a Kong node. The return value is an interface,
+// and needs to be parsed by the caller.
```

Comment:
Can we provide a schema struct that could be used to parse this return value to? I wonder if there isn't a known structure which is already available to be used in here for this purpose.

Reply:
In KIC [`deck/types.Content`](https://github.com/Kong/deck/blob/43ac0a8426b315ab732f81c6423369fb924757be/file/types.go#L705) is used:  https://github.com/Kong/kubernetes-ingress-controller/blob/8d27fd1fa351be9a5e80204a37217a93bea9a5b4/internal/dataplane/sendconfig/inmemory.go#L58

Reply:
I'd suggest making this comment include information similar to what we have for `ReloadDeclarativeRawConfig`: 

```go
// ... should contain the JSON
// serialized body that adheres to the configuration format specified at:
// https://docs.konghq.com/gateway/latest/production/deployment-topologies/db-less-and-declarative-config/#declarative-configuration-format
```

It could also mention the `deck/types.Content` as an example of such a struct for reference.

Reply:
The struct is available in Deck, as @czeslavo pointed out. I updated the comment as you suggested